### ISSUE TITLE
In Vaadin 24 "cancel" was removed from UploadI18N, but "cancel" instance variable still present

### ISSUE THREAD

**Issue by rmuller** [vaadin/flow-components#4473]
### Description

In Vaadin 24 "cancel" was removed from UploadI18N, but "cancel" instance variable still present.

```
public class UploadI18N implements Serializable {
    ...
    private String cancel; // used nowhere, should be removed
    ...
```

### Expected outcome

see description

### Minimal reproducible example

na

### Steps to reproduce

na

### Environment

Vaadin version(s): 24.x

**Comment by web-padawan** [comment]
Thanks for the issue! I missed to remove it indeed, here is a PR with a fix: #4474

**Comment by rmuller** [comment]
Wow, that is fast! Good start of this new year!
